.github/workflows/build.yml

```
@@ -2,80 +2,72 @@
+      - name: Release with artifacts
+        uses: softprops/action-gh-release@v2
         with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: OpenSSH_GUI/bin/Release/net8.0/osx-x64/publish/OpenSSH_GUI
-          asset_name: OpenSSH-GUI-osx-x64.bin
-          asset_content_type: application/x-openssh-gui
+          # Use the output from the publish step to find the files
+          files: |
+            ${{ steps.publish.outputs.artifact_path }}/*
+          # Automatically generate the release body from commits
+          generate_release_notes: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)